src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px auto 0;
+  padding: 0 120px;
+  box-sizing: content-box;
```

Comment:
It is good practice not to change box-sizing. If you set up 'border-box' in reset file, try not to use content-box anywhere